Airports/EGGW/Stars.txt

```diff
@@ -1,40 +1,44 @@
+STAR:EGGW:25:ABBOT1Z:ABBOT TABIS BUSTA LOREL
+STAR:EGGW:07:ABBOT1Z:ABBOT TABIS BUSTA LOREL
 STAR:EGGW:25:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
 STAR:EGGW:07:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:25:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
+STAR:EGGW:07:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
 STAR:EGGW:25:BARMI1A:BARMI MEGEL DITOB LAPRA ABBOT
 STAR:EGGW:07:BARMI1A:BARMI MEGEL DITOB LAPRA ABBOT
 STAR:EGGW:25:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
 STAR:EGGW:07:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:25:DET1A:DET ABBOT
-STAR:EGGW:07:DET1A:DET ABBOT
+STAR:EGGW:25:DET2A:DET LOFFO ABBOT
```

Comment:
Can't see LOFFO defined as a fix anywhere else in the sector file.

Will require adding.